### ISSUE TITLE
Custom entry points do not work in maintenance mode

### ISSUE THREAD

**Issue by aschempp** [contao/core-bundle#713]
The maintenance mode checks for url `/contao` or the *FE preview* cookie. However, custom entry points often live in `system/modules` and are therefore not affected. The do have a `TL_MODE` statement which we could use to trigger non-maintenance.

**Comment by leofeyer** [comment]
You mean more new code to support deprecated legacy stuff? 😄

**Comment by aschempp** [comment]
So I figured out the issue. We're actually checking [if a backend user is logged in](https://github.com/contao/core-bundle/blob/master/src/EventListener/BypassMaintenanceListener.php), but due to our custom `initialize.php` setup of the request, the session cannot be loaded. Therefore the session ID is not correctly compared and the maintenance mode not bypassed.

I tried to quickly rewrite the initialize.php but was unable to get the correct session. Anyone any idea why? …

**Comment by Toflar** [comment]
The request is not created from the globals but instead manually:

```php
$response = $kernel->handle(Request::create('/_contao/initialize', 'GET', [], [], [], $_SERVER));
```

e.g. cookies are missing here ;-)

**Comment by aschempp** [comment]
I tried adding cookies, but that did not work. I even tried so override `$_SESSION['REQUEST_URI']` and then `Request::createFromGlobals`, still the session was not loaded. Everything else worked perfectly :-(

**Comment by leofeyer** [comment]
Fixed in 685b2bfe5732829edda38a61a6b211ccbc656067.
